vpr/src/noc/noc_traffic_flows.cpp

```diff
@@ -0,0 +1,169 @@
+
+#include "noc_traffic_flows.h"
+#include "vpr_error.h"
+
+// constructor indicates that the class has not been constructed yet
+NocTrafficFlows::NocTrafficFlows(void) {
```

Comment:
remove void.
Update comment: default constructor indicates the class hasn't been properly initiatilized yet.

Reply:
done